packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -43,6 +79,17 @@ export class CheckpointService implements TreeDataProvider<BaseNode> {
   public update(): void {
     this._onDidChangeTreeData.fire();
```

Comment:
You can remove the `update` method if you're not using it. I originally put it there to give you a hint on how to trigger the UI update.

Reply:
Done. It'll be gone when I push the update based upon feedback.